src/components/sponsorship/SponsorshipPricingGrid.tsx

```
@@ -0,0 +1,120 @@
+        throw new Error("No checkout URL returned.");
+      }
+    } catch (err: any) {
+      toast.error(err.message || "Failed to start checkout.");
+    } finally {
+      setPurchasingId(null);
+    }
+  };
+
+  const handleContactCustom = () => {
+    window.location.href = `mailto:sponsorships@campusconnect.app?subject=${encodeURIComponent(
+      `Custom sponsorship package for ${clubName || "your club"}`
```

ESLint

[error] Insert `,`

(prettier/prettier)